src/frontend/js/widgets/SyllabusCourseRunsList/SyllabusCourseRun.tsx

```diff
@@ -0,0 +1,117 @@
+    id: 'components.SyllabusCourseRun.languages',
+    description: 'Title of the languages section of an opened course run block',
+    defaultMessage: 'Languages',
+  },
+  runPeriod: {
+    id: 'components.SyllabusCourseRun.enrollmentPeriod',
+    description: 'Enrollment date of an opened course run block',
+    defaultMessage: 'From {startDate} to {endDate}',
+  },
+  coursePeriod: {
+    id: 'components.SyllabusCourseRun.coursePeriod',
+    description: 'Course date of an opened course run block',
+    defaultMessage: 'From {startDate} to {endDate}',
+  },
+});
+
+const OpenedCourseRun = ({ courseRun }: { courseRun: CourseRun }) => {
+  const formatDate = useDateFormat();
+  const intl = useIntl();
+  return (
+    <div className="course-detail__run-descriptions course-detail__run-descriptions--course_and_search">
+      <h3>{courseRun.title}</h3>
+      <dl>
+        <dt>
+          <FormattedMessage {...messages.enrollment} />
+        </dt>
+        <dd>
+          <FormattedMessage
+            {...messages.runPeriod}
+            values={{
+              startDate: formatDate(courseRun.enrollment_start),
+              endDate: formatDate(courseRun.enrollment_end),
+            }}
+          />
+        </dd>
+        <dt>
+          <FormattedMessage {...messages.course} />
+        </dt>
+        {courseRun.start && <meta content={formatDate(courseRun.start)} property="startDate" />}
+        {courseRun.end && <meta content={formatDate(courseRun.end)} property="endDate" />}
+        <dd>
+          <FormattedMessage
+            {...messages.runPeriod}
+            values={{
+              startDate: formatDate(courseRun.start),
+              endDate: formatDate(courseRun.end),
+            }}
+          />
+        </dd>
+        <dt>
+          <FormattedMessage {...messages.languages} />
+        </dt>
+        <dd>
+          {joinAnd(
+            courseRun.languages.map(
+              (language) => intl.formatDisplayName(language, { type: 'language' })!,
+            ),
+            intl,
+          )}
+        </dd>
+      </dl>
+      {findLmsBackend(courseRun.resource_link) ? (
+        <CourseRunEnrollment courseRun={courseRun} context={{} as RichieContext} />
+      ) : (
+        <a className="course-run-enrollment__cta" href={courseRun.resource_link}>
+          {StringHelper.capitalizeFirst(courseRun.state.call_to_action)}
+        </a>
+      )}
+    </div>
+  );
+};
+
+export const SyllabusCourseRun = ({
+  courseRun,
+  course,
+}: {
+  courseRun: CourseRun;
+  course: Course;
+}) => {
+  return (
+    <DjangoCMSTemplate plugin={DjangoCMSPluginCourseRun(courseRun)}>
+      {isJoanieProduct(courseRun) ? (
+        <div className="course-detail__run-descriptions course-detail__run-descriptions--course_and_search">
+          <CourseProductItem productId={extractResourceId(courseRun)!} courseCode={course.code!} />
+        </div>
+      ) : (
+        <OpenedCourseRun courseRun={courseRun} />
+      )}
```

Comment:
Why not ? .. then remove the corresponding div from `OpenedCourseRun` component.

```suggestion
        <div className="course-detail__run-descriptions course-detail__run-descriptions--course_and_search">
      {isJoanieProduct(courseRun) ? (
          <CourseProductItem productId={extractResourceId(courseRun)!} courseCode={course.code!} />
      ) : (
        <OpenedCourseRun courseRun={courseRun} />
      )}
      </div>
```